src/rviz_video_encoder/video_encoder.c

```diff
@@ -140,8 +140,7 @@ VideoEncoder* video_encoder_init(VideoEncodeParams params){
     enc->cc->time_base = enc->st->time_base;
     enc->cc->pix_fmt = enc->frame_scaled->format;
 
-    // RVN:TODO this is good neighboor mode. probably should have drag race mode
-    //enc->cc->thread_count = 1;
+    enc->cc->thread_count = enc->params.thread_count;
```

Comment:
For posterity: this sets thread count for threads interacting with the codec (copying in frames, evacuating output buffers, etc). This mainly relates IO timing/contention between codec, input (rviz frame scaler from display buffer) and output (mp4 muxer to disk).

Lines 162-263 below in this diff set the number of threads used by the codec internally to do computationally expensive encoding tasks.